### DISCUSSION TITLE
Some markdown rendering not working well on Chrome

### DISCUSSION THREAD

**Discussion by moggieuk** [community/community#61553] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I've created markdown documentation of a project and one page indicated a problem rendering on Chrome browser (not top level README.md and on Mac).  It seems to be provoked when you have several block quote spaces on one page.  The page renderers and behaves perfectly in preview tab but when pushed, the resultant page corrupts not only its frame contents but the file navigation tree on the left. Simply scroll up and down and usually the navigation tree on the left disappears and there are gaps in the rendering of the content frame.

The page renders perfectly in Safari.

Form of page is as follows, but real page is [here](https://github.com/moggieuk/Happy-Hare/blob/main/doc/configuration.md)

```yml
foo: bar
```

Some text

```yml
foo: bar
```

Some text

```yml
foo: bar
```

Some text

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @moggieuk ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
